### DISCUSSION TITLE
Welcome to eudi-doc-architecture-and-reference-framework Discussions!

### DISCUSSION THREAD

**Discussion by digeorgi** [eu-digital-identity-wallet/eudi-doc-architecture-and-reference-framework#167] [Announcements]
📣 Attention Community Members! Let's utilize the 'Discussions' to elaborate on topics towards improving the Architecture Reference Framework!

👋 Hello everyone,

We're excited to introduce a new way for us to collaborate and discuss improvements to our document repository! We've enabled the Discussions feature on our GitHub repository to provide a dedicated space for all of us to share ideas, ask questions, and propose enhancements to our Markdown documents.

🤔 Why Use Discussions?

The Discussions feature offers several benefits:

Centralized Communication: Keep all discussions and ideas in one place, making it easier for everyone to stay informed and engaged.
Collaborative Problem-Solving: Collaborate with fellow community members to address questions, solve problems, and suggest improvements to our documents.
Transparent Decision-Making: Ensure transparency by discussing proposed changes openly, gathering feedback, and making informed decisions as a community.
Accessible Archives: Discussions are archived and searchable, allowing new members to catch up on past conversations and contributions easily.
💡 How Can You Get Involved?

Share Your Ideas: Have an idea for improving our documents? Share it in Discussions! Whether it's a suggestion for new content, clarifications, or formatting improvements, we want to hear from you.
Ask Questions: Stuck on something? Need clarification on a topic? Don't hesitate to ask questions in Discussions. Chances are, someone else might have the same question, and together, we can find the answers.
Contribute Your Expertise: Are you an expert in a particular area covered by our documents? Share your insights and expertise to help improve the quality and accuracy of our content.
🌟 Let's Collaborate!

Our community thrives on collaboration and shared knowledge, and Discussions provide an excellent platform for us to work together towards our common goals. 

🔗 Access Discussions: [Link to Discussions](https://github.com/eu-digital-identity-wallet/eudi-doc-architecture-and-reference-framework/discussions)

If you have any questions or need assistance with using the Discussions feature, feel free to reach out to our moderators or the project maintainers.

Thank you for your continued support and contributions to our open-source project!

Happy discussing! 🚀

**Comment by jantdm** [comment]
Thank you, I think it makes sense to have this. I have an Issue open that is more of a discussion for now: issue #159. Unfortunately, I'm missing the link to convert it to a discussion as described here: https://docs.github.com/en/discussions/managing-discussions-for-your-community/moderating-discussions. Can a maintainer/moderator please convert it to a discussion? Or maybe there is a general issue-to-discussion conversion setting/flag that needs to be set (the reverse discussion to issue is possible)? Please let me know and if it can't be converted and I will copy-paste it.

**Reply by skounis** [reply]
Hi @jantdm it has been moved: https://github.com/eu-digital-identity-wallet/eudi-doc-architecture-and-reference-framework/discussions/168?converting=1

**Comment by sander** [comment]
When to best use Issues and when to use Discussions?

**Reply by digeorgi** [reply]
Thank you for your question. In summary, GitHub Discussions are ideal for open-ended conversations, brainstorming, and community engagement, while GitHub Issues are more suitable for addressing specific problems, tracking actionable items, and managing feature requests or documentation improvements. 

Use **GitHub Discussions** when:
1. **General Discussion:** You want to engage in open-ended discussions, share ideas, or gather feedback from the community without a specific actionable item or problem to address.
2. **Brainstorming and Collaboration**: You're looking to brainstorm ideas, collaborate on new features or enhancements, or seek input on broader topics related to the project.
3. **Questions and Help**: You have questions about how to use the project, need clarification on certain topics, or require assistance from other community members.
4. **Announcements and Updates**: You want to make project announcements, share updates, or provide general information to the community.

Use **GitHub Issues** when:
1. **Specific Issue or Problem**: You've identified a specific problem, typo, or error within the project that needs to be addressed by the maintainers.
2. **Feature Requests and Enhancements**: You have a specific feature request or enhancement suggestion that you'd like to propose for consideration by the project maintainers.
3. **Documentation Errors or Improvements**: You've found errors in the documentation, such as typos, inaccuracies, or missing information, that need to be corrected.
4. **Actionable Items**: You have a specific actionable item or task that needs to be assigned, tracked, and resolved by the ARF writing group.
5. **Technical Discussions**: You need to discuss technical details, implementation strategies, missing angles that require a focused and structured approach.

**Reply by digeorgi** [reply]
No worries though, in case you are undecisive you can use either and can be always transformed to the most appropriate type.
